analyses/fusion_filtering/02-fusion-filtering.R

```diff
@@ -63,17 +63,27 @@ option_list <- list(
 
 opt <- parse_args(OptionParser(option_list=option_list))
 
-standardFusionFile<-opt$standardFusionFile
+# TO-DO
+# multiple opt values for each caller and output name from input fusion calls and expression Matrix?
+standardFusionFiles<-unlist(strsplit(opt$standardFusionFiles,","))
 expressionMatrix<-opt$expressionMatrix
-reathroughFilter<-opt$reathroughFilter
+readthroughFilter<-opt$readthroughFilter
 expressionFilter<-opt$expressionFilter
 artifactFilter<-opt$artifactFilter
 junctionReadCountFilter<-opt$junctionReadCountFilter
 spanningFragCountFilter<-opt$spanningFragCountFilter
 referenceFolder<-opt$referenceFolder
 readingFrameFilter<-opt$readingFrameFilter
 
-standardFusioncalls<-readr::read_tsv(standardFusionFile)
+# standardFusioncallsSTARFusion<-readr::read_tsv(standardFusionFileSTARFusion)
+# standardFusioncallsArriba<-readr::read_tsv(standardFusionFileArriba)
+# 
+# # combine callers
+# standardFusioncalls<-rbind(standardFusioncallsArriba,standardFusioncallsSTARFusion)
+
+standardFusioncalls<-lapply(standardFusionFiles,function(x) readr::read_tsv(x))
+standardFusioncalls <- plyr::ldply(standardFusioncalls, data.frame) 
```

Comment:
This will only do rbind if the resulting dataframes have the same columns like we have here from standardization step
standardFusioncalls<-lapply(standardFusionFiles,function(x) readr::read_tsv(x))
> nrow(standardFusioncalls[[1]])
[1] 7515
> nrow(standardFusioncalls[[2]])
[1] 54239

standardFusioncalls <- plyr::ldply(standardFusioncalls, data.frame) 
nrow(standardFusioncalls)
[1] 61754